tests/unit/test_client.py

```diff
@@ -7839,7 +7835,6 @@ def test_load_table_from_dataframe_unknown_table(self):
         )
 
     @unittest.skipIf(pandas is None, "Requires `pandas`")
-    @unittest.skipIf(fastparquet is None, "Requires `fastparquet`")
     def test_load_table_from_dataframe_no_pyarrow_warning(self):
```

Comment:
I'm a bit surprised to see this test passing. I guess we still have some code that falls back to the default pandas parquet rendering?

Can you look into if we can remove that code path?

Related: We should be able to simplify this docstring now: https://github.com/googleapis/python-bigquery/blob/cbcb4b86d2d01fb983c20271958b11665e8f7d03/google/cloud/bigquery/client.py#L2134-L2147